hathor/cli/nginx_config.py

```diff
@@ -240,11 +240,12 @@ def generate_nginx_config(openapi: dict[str, Any], *, out_file: TextIO, rate_k:
 
     server_open = f'''
 upstream backend {{
-    server fullnode:8080;
+    server 127.0.0.1:8080;
 }}
 
 server {{
     listen 80;
+    listen [::]:80;
```

Comment:
Should we add an `--enable-ipv6` here too?

Reply:
Actually, now that you mentioned, I'm wondering whether we actually need to add this here.

I had added it mostly to test IPv6 support in the API. But in our deploys, we won't really need it, because the API is server through a Load Balancer. The LB will still talk IPv4 with Nginx.

I think we could even remove it and it would make no difference.

However, I guess there is no harm in leaving it and always making it listen in the IPv6 interface, is there? The image would be ready for anyone that needs it eventually over IPv6, for any reason. And if it's not needed, it just won't be used.

Another thing is that we would be creating more variations of the Nginx Docker image that we generate. We already have the `latest` and `no-rate-limit-latest`.

We would probably need to generate additionally the `latest-with-ipv6` and `no-rate-limit-latest-with-ipv6`.

I would rather avoid these variations unless we have a good reason to not always listen on IPv6 in Nginx.

Reply:
I agree that there's no harm in enabling it. Let's keep it.